update-version.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#Execute this script from local checkout of spring cloud stream
```

Comment:
What is this script about?
Do we really need it checked out to the public repo?